test-suite/Makefile

```diff
@@ -250,11 +250,11 @@ $(addsuffix .log,$(wildcard bugs/closed/*.v)): %.v.log: %.v
 # Unit tests
 #######################################################################
 
-OCAMLOPT := $(OCAMLFIND) opt $(CAMLFLAGS)
-SYSMOD:=-package num,str,unix,dynlink,threads
+ifeq ($(LOCAL),true)
+  export OCAMLPATH := $(patsubst "%",%,$(COQLIBINSTALL))
+endif
```

Comment:
Why isn't that test around setting `LIB` and then an unconditional `export OCAMLPATH := ...`?

Reply:
This breaks local building of the test suite:

```
OCAMLOPT  unit-tests/src/utest.mli
ocamlfind: Package `oUnit' not found
```

Reply:
I see what happened, with the latest change we are resetting the `OCAMLPATH` instead of appending to it, thus Nix breaks. Does:
```
ifeq ($(LOCAL),true)
  export OCAMLPATH := $(shell echo $(COQLIBINSTALL):$$OCAMLPATH)
endif
```
work? [The quoting stuff is crazy with `make`]

Reply:
In fact I knew of this problem but somehow the proper rule got lost in the last commit.

Reply:
Works for me. Thanks.

Reply:
Do you submit the PR? Otherwise I can do it.

Reply:
Ok I got the PR ready so don't bother.